package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eeacms/volto-plotlycharts",
-  "version": "4.0.0",
```

Comment:
@andreiggr As this doesn't seems to be a patch PR, Please bump version to 4.1.0 or 5.0.0

Reply:
Done